src/libstd/time/duration.rs

```diff
@@ -188,6 +188,13 @@ impl DivAssign<u32> for Duration {
     }
 }
 
+#[stable(feature = "duration_default", since = "1.9.0")]
+impl Default for Duration {
+    fn default() -> Duration {
+        Duration::new(0, 0)
```

Comment:
Yeah, that was exactly the argument I made in the reddit comment. And of course the default just has to make intuitive sense; we cannot foresee possible applications. Still, I'd like to have more input for this before we commit to an implementation, especially as insta-stability means we cannot easily back off.